front/app/components/FullscreenImage.tsx

```diff
@@ -57,7 +57,7 @@ const FullscreenImage = ({ src, altText }: Props) => {
       justifyContent="center"
       alignItems="center"
     >
-      <Box position="relative" w="100%">
+      <Box position="relative">
```

Comment:
Needed to remove this to keep the zoom button inside the picture (if the picture was not as wide as the selection box)
<img width="270" alt="Screenshot 2024-05-22 at 16 44 02" src="https://github.com/CitizenLabDotCo/citizenlab/assets/16427929/484e2582-967c-436f-902c-f7d2f070d13a">